### PR TITLE
Add expires_at to bindings table in db

### PR DESCRIPTION
**Pull request by MarekMichali**
**Description**

This change is a result of a meeting where we decided to add the expires_at file to the DB to avoid to calculate this value every time we need it.

Changes proposed in this pull request:

- Add expires_at to DB
- Calculate and insert this value to the DB when creating bindings

**Related issue(s)**

See #1198

### CODE REVIEW

File: internal/broker/bind_create.go [hunk 1]
```diff
@@ -123,6 +123,7 @@ func (b *BindEndpoint) Bind(ctx context.Context, instanceID, bindingID string, d
 		InstanceID: instanceID,
 
 		CreatedAt: time.Now(),
+		ExpiresAt: time.Now().Add(time.Duration(int64(expirationSeconds)) * time.Second),
```
Comment (jaroslaw-pieszka) [inline internal/broker/bind_create.go:126]: Are we sure the time of creation of this object is the same as the time of creation of the token?

File: internal/fixture/fixture.go [hunk 2]
```diff
@@ -356,6 +356,7 @@ func FixBindingWithInstanceID(bindingID string, instanceID string) internal.Bind
 
 		CreatedAt: time.Now(),
 		UpdatedAt: time.Now().Add(time.Minute * 5),
+		ExpiresAt: time.Now().Add(time.Second * 600),
```
Comment (jaroslaw-pieszka) [inline internal/fixture/fixture.go:359]: Please consider comment, there are people who do not get that 600 seconds is equal to 10 minutes.

File: internal/broker/bind_create.go (hunk 1 above)
Comment (MarekMichali) [reply internal/broker/bind_create.go:126]: I'm not sure, I will update this to check the time returned in the token.

**Review by jaroslaw-pieszka — APPROVED** [review]

**Pull request merged**